consensus/event_loop.go

```diff
@@ -782,6 +782,7 @@ func (fc *fastHotStuffEventLoop) tryConstructTimeoutQCInCurrentView() *FastHotSt
 		View:           fc.currentView,                          // The view that the timeout QC is proposed in
 		TipBlockHash:   validatorsHighQC.GetBlockHash(),         // The block hash that we extend from
 		TipBlockHeight: safeBlock.GetHeight(),                   // The block height that we extend from
+		QC:             validatorsHighQC,                        // The high QC aggregated from the timeout messages
```

Comment:
Are we expecting the QC field to be populated for a timeout QC? we already have the high QC in the AggregateQC struct.

Reply:
Correct. The QC field is populated in the timeout QC event just for convenience. The reference implementation populates it too for convenience. This way, the QC field just tells us the high QC that we will extend the next block from.

The QC field in the final proposed block won't be populated though